src/bp/core/services/dialog/instruction/strategy.ts

```diff
@@ -9,29 +9,14 @@ import { inject, injectable, tagged } from 'inversify'
 import _ from 'lodash'
 import { NodeVM } from 'vm2'
 
-import { container } from '../../../app.inversify'
 import { renderTemplate } from '../../../misc/templating'
 import { TYPES } from '../../../types'
 import { VmRunner } from '../../action/vm'
 
-import { Instruction, InstructionType, ProcessingResult } from '.'
+import { Instruction, ProcessingResult } from '.'
 
 const debug = DEBUG('dialog')
 
-@injectable()
-export class StrategyFactory {
```

Comment:
Was that used anywhere else than the `InstructionProcessor` singleton ? 
if not I'm trying to understand what that existed in the first place > <

Reply:
Don't ask...